runtime/autoload/tar.vim

```diff
@@ -34,9 +35,9 @@ if &cp || exists("g:loaded_tar")
  finish
 endif
 let g:loaded_tar= "v32b"
-if v:version < 702
+if v:version < 900
  echohl WarningMsg
- echo "***warning*** this version of tar needs vim 7.2"
+ echo "***warning*** this version of tar needs vim 9.0"
```

Comment:
`tar.vim` uses `str2blob()` and `blob2str()`, which are both introduced after Vim 9.1 release.